task/taskutils.go

```diff
@@ -0,0 +1,3 @@
+package task
+
```

Comment:
hmm I think you need to rebase from main, this is defined in `task/version_checker.go`, with legacy and new version